decode/render.ts

```diff
@@ -113,12 +115,12 @@ function css(style: CSSStyleDeclaration, field: string): number {
 
 export function markup(type: Event, data: DomData[], iframe: HTMLIFrameElement): void {
     let doc = iframe.contentDocument;
-    if (type === Event.Discover) { reset(); }
```

Comment:
Is this the change for the first payload missing?

Reply:
No, this is all playback (not mainline instrumentation code). See comment above for explanation.